docs/src/pages/components/badges/CustomizedBadges.tsx

```diff
@@ -9,9 +9,9 @@ import ShoppingCartIcon from '@material-ui/icons/ShoppingCart';
 const StyledBadge1 = withStyles((theme: Theme) =>
   createStyles({
     badge: {
-      top: '50%',
```

Comment:
My objective was to reproduce https://store.google.com/ cart: 
![Capture d’écran 2019-08-31 à 23 41 25](https://user-images.githubusercontent.com/3165635/64069384-dd0f7480-cc48-11e9-99c4-b2a23888e387.png)

Reply:
Every time I saw it, I thought "no designer would do that". Guess I was wrong! 😆

Reply:
Haha, I'm happy either way, wanted to raise the context.

Reply:
I'd already reverted it before I saw this, so let's leave it how you intended.